tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "strict": true,
     "baseUrl": "./packages",
     "skipLibCheck": true,
-    "moduleResolution": "node",
+    "moduleResolution": "nodenext",
```

Comment:
Needed to load `@ecies/ciphers/aes`. The value `node` is deprecated anyway, we shouldn't use it, see https://www.typescriptlang.org/tsconfig/#moduleResolution